README.md

```diff
@@ -38,44 +38,64 @@ Follow installation instructions [here](https://pygraphviz.github.io/documentati
 
 ### Knowledge graph construction
 
+**NOTE: The [APOC core library](https://neo4j.com/labs/apoc/) must be installed in your Neo4j instance in order to use this feature**
+
+Assumption: Neo4j running
+
 ```python
+from neo4j import GraphDatabase
+from neo4j_graphrag.embeddings import OpenAIEmbeddings
 from neo4j_graphrag.experimental.pipeline.kg_builder import SimpleKGPipeline
 from neo4j_graphrag.llm.openai_llm import OpenAILLM
 
+# Connect to Neo4j database
+URI = "neo4j://localhost:7687"
+AUTH = ("neo4j", "password")
+driver = GraphDatabase.driver(URI, auth=AUTH)
+
 # Instantiate Entity and Relation objects
-entities = ["PERSON", "ORGANIZATION", "LOCATION"]
-relations = ["SITUATED_AT", "INTERACTS", "LED_BY"]
+entities = ["Book", "Genre", "Person", "Date"]
+relations = ["AUTHORED_BY", "CATEGORIZED_BY", "PUBLISHED_ON"]
 potential_schema = [
-    ("PERSON", "SITUATED_AT", "LOCATION"),
-    ("PERSON", "INTERACTS", "PERSON"),
-    ("ORGANIZATION", "LED_BY", "PERSON"),
+    ("Book", "AUTHORED_BY", "Person"),
+    ("Book", "CATEGORIZED_BY", "Genre"),
+    ("Book", "PUBLISHED_ON", "Date"),
```

Comment:
Do you think this example could be misinterpreted as "we can't create node properties"? Because year could also be a property of `Book`, which is possible to do with the package but not with this simple definition yet.

Reply:
I think this is alright for now. We could perhaps introduce a more advanced Pipeline constructor later on to make this clearer, wdyt?

Reply:
Make sense. I'm happy to try another example though if this one is a bit confusing

Reply:
Maybe it's not controversial, but I'd prefer to validate that with someone.